src/streaming/streamingclient.py

```diff
@@ -0,0 +1,132 @@
+"""StreamingClient tool used for streaming services"""
+from AudioConfig import AudioConfig
+import websocket
+import io
+import threading
+import six
+import json
+
+def on_error(error):
+    raise error
+
+def on_close(code, reason):
+    print("{} : {}".format(code, reason))
```

Comment:
We should add a short message here like for connected. Something like `Connection Closed with {code}: {reason}`